### ISSUE TITLE
User registration endpoint is missing

### ISSUE THREAD

**Issue by yipivan** [sacconazzo/directus-extension-api-docs#37]
The default registration entry `/users/register` endpoint is missing from the Swagger Docs.
https://docs.directus.io/reference/authentication.html#request

**Comment by sacconazzo** [comment]
Yes, I noticed this, at the moment the `/users/register` endpoint is not even present in the standard openapi specification (`/server/specs/oas`) with admin authentication either. Even if in fact it is a present and usable endpoint.

I have not investigated why Directus does not include the endpoint or will only include it in the future.

If it were also included in the standard openapi, it would consequently also be inherited by the extension (example: `/api-docs/oas`) together with the defined custom endpoints.
